BigInt_test.c

```diff
@@ -63,7 +67,7 @@ void BigInt_test_basic() {
     BigInt_test_operations(1000, 1);
     BigInt_test_operations(2546, 2546);
     BigInt_test_operations(1234, 4321);
-    BigInt_test_operations(999999999, 123456789);
+    //BigInt_test_operations(999999999, 123456789);
```

Comment:
```suggestion
    BigInt_test_operations(999999999, 123456789);
```
Looks like this last test still fails. I haven't had time to dig into it further yet but probably best to understand it before merging.

```
Testing basic 2-argument operations
Assertion failed: (BigInt_to_int(big_int_a, &result)), function BigInt_test_single_operation, file BigInt_test.c, line 172.
```

Reply:
The test is failing because it can't be converted back to int. 999999999 * 123456789 doesn't fit into a 32-bit int.

Reply:
Ah, I see.